tests/features/consumer/searchPostDocumentReference-success.feature

```diff
@@ -174,6 +211,34 @@ Feature: Consumer - searchDocumentReference - Success Scenarios
       | author      | X26                                   |
     And the Bundle does not contain a DocumentReference with ID 'X26-1111111111-SearchMultipleRefTestDifferentType'
 
-# No pointers found
-# Pointers exist but no permissions
-# Search by custodian
+  Scenario: Search for a DocumentReference by NHS Number where 0 pointers are found
+    Given the application 'DataShare' (ID 'z00z-y11y-x22x') is registered to access the API
+    And the organisation 'DK94' is authorised to access pointer types:
+      | system                 | value           |
+      | http://snomed.info/sct | 736253002       |
+      | http://snomed.info/sct | 861421000000109 |
+    And the organisation 'RX898' is authorised to access pointer types:
+      | system                 | value           |
+      | http://snomed.info/sct | 736253002       |
+      | http://snomed.info/sct | 861421000000109 |
+    And a DocumentReference resource exists with values:
+      | property    | value                          |
+      | id          | DK94-111-DeleteDocRefTest1     |
```

Comment:
is this a deletion test, if so does it belong in the delete feature? I see that it searches after the deletion so it's more of a combined scenario, this could be reflected in the scenario title if it lives here (it's slightly more complex than simply searching by NHS number where there never were any pointers for that patient)

Reply:
Perhaps the deletion is not required, but the main goal of this test is to setup an NHS number which has 0 pointers, so I figured one way to do that is to create a pointer for an NHS number and then delete that pointer. May not be necessary.